src/buttons/CopyToClipboard/styles.css

```diff
@@ -0,0 +1,17 @@
+.toclipboard {
+    font: var(--default-m);
+    font-weight: 500;
+  }
+  
+  .tocopy {
```

Comment:
(nit) please fix the spacing in this file, it appears that all classes after `. toclipboard` have been indented when they should align with `.toclipboard {`

also, please consider renaming by adding a hyphen for improved readability, like so: 

`.to-clipboard {`

and 

` .to-copy {`